shell/platform/common/client_wrapper/testing/stub_flutter_api.cc

```diff
@@ -93,6 +95,31 @@ void FlutterDesktopMessengerSetCallback(FlutterDesktopMessengerRef messenger,
   }
 }
 
+FlutterDesktopMessengerRef FlutterDesktopMessengerAddRef(
+    FlutterDesktopMessengerRef messenger) {
+  assert(false);  // not implemented
+  return nullptr;
```

Comment:
No existing tests for the client_wrapper actually hit this code.  In terms of code coverage it isn't strictly needed since I have coverage in the windows unit tests.